client/src/components/TermsAndConditionsModal.js

```diff
@@ -0,0 +1,308 @@
+      }
+    },
+    "& p": {
+      fontSize: "16px",
+      lineHeight: "25px",
+      textAlign: "left",
+      "& img": {
+        margin: "4px 12px 12px 0"
+      }
+    },
+    "& .smaller": {
+      fontSize: "12px",
+      lineHeight: "25px",
+      textAlign: "left"
+    },
+    "& input": {
+      boxSizing: "border-box",
+      fontSize: "20px",
+      lineHeight: "24px",
+      padding: "16px",
+      border: "1px solid #979797",
+      marginTop: "8px"
+    },
+    "& .scroll": {
+      overflowY: "scroll",
+      width: "auto",
+      height: "300px"
+    }
+  },
+  modalActions: {
+    display: "flex",
+    justifyContent: "flex-end",
+    marginTop: "42px",
+    "& button": {
+      fontFamily: "Calibri Bold",
+      letterSpacing: "2px",
+      height: "60px",
+      display: "inline",
+      marginLeft: "auto",
+      marginRight: "auto",
+      border: "solid",
+      fontSize: "20px",
+      lineHeight: "24px",
+      textAlign: "center",
+      cursor: "pointer",
+      textTransform: "uppercase"
+    }
+  },
+  acceptBtn: {
+    width: "150px",
+    height: "50px",
+    marginLeft: "10px",
+    marginRight: "auto",
+    backgroundColor: "#A7C539",
+    color: "#000000",
+    boxShadow: "0px 6px 4px rgba(0, 46, 109, 0.3)"
+  },
+  declineBtn: {
+    width: "300px",
+    height: "50px",
+    marginRight: "10px",
+    marginLeft: "auto",
+    backgroundColor: "transparent",
+    color: "rgba(0, 0, 0, 0.5)"
+  },
+  doNotShowBtn: {
+    border: "1px solid black",
+    margin: "2px",
+    width: "20px",
+    height: "20px !important",
+    position: "relative",
+    marginRight: "10px !important"
+  }
+});
+
+const modalStyles = {
+  overlay: {
+    backgroundColor: "rgba(0, 0, 0, 0.05)",
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center"
+  },
+  content: {
+    position: "relative",
+    top: "50px",
+    right: "auto",
+    bottom: "auto",
+    left: "200px",
+    boxSizing: "border-box",
+    height: "600px",
+    maxHeight: "100%",
+    width: "800px",
+    maxWidth: "100%",
+    padding: "60px",
+    backgroundColor: "#ffffff",
+    boxShadow: "0px 5px 10px rgba(0, 46, 109, 0.2)",
+    overflow: "scroll"
+  }
+};
+
+const TermsAndConditionsModal = () => {
+  const classes = useStyles();
+  const [
+    termsAndConditionsModalOpen,
+    setTermsAndConditionsModalOpen
+  ] = useState(true);
+  const history = useHistory();
+
+  const toggleTermsAndConditionsModal = () => {
+    setTermsAndConditionsModalOpen(!termsAndConditionsModalOpen);
+  };
+
+  if (localStorage.getItem("termsAndConditions")) return null;
+
+  return (
+    <Modal
+      isOpen={termsAndConditionsModalOpen}
+      onRequestClose={toggleTermsAndConditionsModal}
+      shouldCloseOnOverlayClick={false}
+      contentLabel="Terms and Conditions Modal"
+      style={modalStyles}
+      className={classes.modal}
+    >
+      <h2>TDM Calculator User Terms and Conditions</h2>
+      <div className="scroll">
```

Comment:
classname should be {classes.scroll} once you put `scroll` as the key in the useStyles